src/lib/helpers/erc725.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { getERC725 } from './erc725';
+
+const randomAddress = '0xc0ffee254729296a45a3885639AC7E10F9d54979';
+describe('erc725.js', () => {
+  it('should be configured with the address if available', () => {
+    const erc725Helper = getERC725(randomAddress);
+    expect(erc725Helper.options.address).toEqual(randomAddress);
+  });
+});
```

Comment:
Why do you set `randomAddress` outside of the test case?
Do you want to use it later in other test cases?

Reply:
yeah, we might add more test cases, but either way is fine for me.